argocd-boostrap/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+echo "checking ArgoCD version on the server"
+argocd version --short --client  
+
+echo "change argocd-server service type to load balancer"
+export ARGO_SERVER=$(kubectl get svc -n argocd -l app.kubernetes.io/name=argocd-server -o name) 
+
+echo "set argocd password as env variable" 
+export ARGO_PASSWORD=$(kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d)
+
+echo "login to the server and change password"
+argocd login $ARGO_SERVER --username admin --password $ARGO_PASSWORD 
+
+echo "bootstrap app of apps"
+argocd app create apps --dest-namespace argocd  --dest-server https://kubernetes.default.svc  --repo git@github.com:shapirov103/argo-apps.git --path "."
```

Comment:
You don't need a github token for this to work? What if it is a private repo?

Reply:
added steps based on argo documentation to address this - https://argoproj.github.io/argo-cd/user-guide/private-repositories/#credential-templates